policy_set.go

```diff
@@ -71,9 +74,18 @@ type PolicySet struct {
 	UpdatedAt      time.Time `jsonapi:"attr,updated-at,iso8601"`
 
 	// Relations
+	// The organization to which the policy set belongs to.
 	Organization *Organization `jsonapi:"relation,organization"`
-	Policies     []*Policy     `jsonapi:"relation,policies"`
-	Workspaces   []*Workspace  `jsonapi:"relation,workspaces"`
+	// The workspaces to which the policy set applies.
+	Workspaces []*Workspace `jsonapi:"relation,workspaces"`
+	// Individually managed policies which are associated with the policy set.
+	Policies []*Policy `jsonapi:"relation,policies"`
+	// The most recently created policy set version, regardless of status.
+	// Note that this relationship may include an errored and unusable version,
+	// and is intended to allow checking for errors.
+	PolicySetNewestVersion *PolicySetVersion `jsonapi:"relation,newest-version"`
```

Comment:
It looks like this went through as `PolicySetNewestVersion` rather than the suggested `NewestVersion` since the context is within the type already, i.e., PolicySets. It looks odd to type the type.

Reply:
:doh: I read Chris's [comment](https://github.com/hashicorp/go-tfe/pull/234#discussion_r651923824) as to swap the `VersionNewest` with `NewestVersion`. I didn't read it as replace `PolicySetVersionNewest` with _just_ `NewestVersion`.

I'll open up a PR for this.

Reply:
PR: https://github.com/hashicorp/go-tfe/pull/235